.github/workflows/doxygen.yml

```diff
@@ -37,15 +37,3 @@ jobs:
       - name: copy html to docs/doxygen_output branch
         shell: bash
         run: cp -r master/documents/html doxygen/documents/
-      - name: GIT commit and push docs
-        env:
-          USER_NAME: github-bot
-          GITHUB_TOKEN: ${{secrets.GITHUB_TOKEN}}
-        run: |
-          git config --local user.name $USER_NAME
-          git config --local user.email "in@valid.email"
-          git add -f doxygen/documents/html
-          git commit -m "update docs"
-          git push -f https://$USER_NAME:$GITHUB_TOKEN@github.com/KratosMultiphysics/Kratos.git docs/doxygen_output
```

Comment:
As a side note, we removed the hability to do this in the master branch, even if the action works the master is gonna reject the push. 

Not even admin accounts can force push directly.

Reply:
Thanks @roigcarlo ... Anyways, the uploading this to a branch in Kratos repo is no go, since the html files size is 5.4 gb. We are looking for other alternatives.